src/client.ts

```diff
@@ -1055,6 +1055,26 @@ export class StreamChat<
     }
   }
 
+  _buildSort(
+    // todo: generalize type
+    sort: UserSort | ChannelSort | ChannelSort[] | UserSort[],
+  ): Array<{ field: string; direction?: AscDesc }> {
```

Comment:
Couldn't come up with generalized type that works for `sort`. Would appreciate the ideas